src/httpsRequestTo.js

```diff
@@ -0,0 +1,21 @@
+const https = require('https');
```

Comment:
httpsRequestTo.js To is redundant

Reply:
Does it have to be a part of the service?